### PR TITLE
Code examples

### PR DESCRIPTION
**Pull request by jchavez443**

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-Transcribe-endpoints-csharp-java.s3-website-us-east-1.amazonaws.com/

File: voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md [hunk 1]
```diff
@@ -54,13 +54,18 @@ curl -X GET \
...
+System.out.println(list.get(0).getText());
 ```
 
 {% sample lang="csharp" %}
 
 ```csharp
-// coming soon
+var response = controller.GetRecordingTranscription(accountId, callId, recordingId);
+
+Console.WriteLine(response.Data.Transcripts[0].Text);n
```
Comment (jmulford-bw) [inline voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md]: extra character `n` at the end

Also maybe add a `.Confidence` too if you want

File: voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md [hunk 2]
```diff
@@ -54,13 +54,18 @@ curl -X GET \
 {% sample lang="java" %}
 
 ```java
-// coming soon
+ApiResponse<TranscriptionResponse>  response = controller.getRecordingTranscription(accountId, callId, recordingId);
+
+List<Transcript> list = response.getResult().getTranscripts();
+System.out.println(list.get(0).getText());
```
Comment (jmulford-bw) [inline voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md:60]: Maybe add `getConfidence()` too?

**Review by jmulford-bw — CHANGES_REQUESTED** [review]

**Review by jmulford-bw — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-Transcribe-endpoints-csharp-java.s3-website-us-east-1.amazonaws.com/

**Pull request merged**